app/helpers/application_helper.rb

```diff
@@ -7,6 +7,17 @@ module ApplicationHelper
   include StiRoutingHelper
   include TextualSummaryHelper
 
+  def icon_text(i, text = "", opts = {})
```

Comment:
I feel like this method already exists somewhere, but I'm not sure.  @epwinchell @h-kataria ?